packages/generate-html/src/index.ts

```diff
@@ -192,9 +224,13 @@ export async function generateHtml(
     throw new Error(`Invalid input type : ${typeof certificateInput}`);
   }
 
-  const { certificate } = castCertificate(rawCert);
+  const { certificate, type } = castCertificate(rawCert);
   const certificateLanguages = getCertificateLanguages(certificate) || ['EN'];
 
+  const externalStandards: ExternalStandards[] =
+    schemaToExternalStandardsMap[type].map((schemaType) => get(certificate, schemaType)) ||
+    [].filter((externalStandards) => externalStandards);
```

Comment:
`[].filter((externalStandards) => externalStandards);` ?

What is it supposed to do ?